help-on-help/writing-help/writing-content-checklist.md

```diff
@@ -0,0 +1,65 @@
+# Doc content checklist
```

Comment:
This add really shouldn't be in this PR. Please create new PRs for new content. I'm not sure I understand these questions. I can answer them, but how does that help guide me on the pages? There needs to be some meaning to "if the answer is yes here, that means XYZ'. 

Please do not address this here or now. The priority is rewriting content. Return to this later